### DISCUSSION TITLE
Can we use the LTC node as well like it is explained here for BTC (ElectrumX)?

### DISCUSSION THREAD

**Discussion by sla-te** [btcpayserver/btcpayserver#3115] [Technical Support] [answered]
https://docs.btcpayserver.org/ElectrumX/#how-to-integrate-electrumx-into-your-btcpay-server-and-connect-your-electrum-wallet-to-it-for-your-complete-privacy

Or what is a suggestable way to handle the LTC wallet?

**Comment by Kukks** [comment, marked as answer]
> [docs.btcpayserver.org/ElectrumX/#how-to-integrate-electrumx-into-your-btcpay-server-and-connect-your-electrum-wallet-to-it-for-your-complete-privacy](https://docs.btcpayserver.org/ElectrumX/#how-to-integrate-electrumx-into-your-btcpay-server-and-connect-your-electrum-wallet-to-it-for-your-complete-privacy)
> 
> Or what is a suggestable way to handle the LTC wallet?

Only if someone bothers to write the docker image + add the option to btcpay-docker. We do not work on altcoin features.

**Reply by pavlenex** [reply]
As to what is suggested to handle altcoin, probably a hot wallet within btcpay itself.

**Reply by sla-te** [reply]
Thanks for the replies, appreciate it.

| As to what is suggested to handle altcoin, probably a hot wallet within btcpay itself.

I haven't described it clear enough, I am indeed using a hot wallet on the server. What I was referring to was the way to handle the wallet and its cashouts itself. Wouldn't it make sense to expose the litecoin node, to be able to use it in Electrum LTC, analogue to how we can do it with ElectrumX?

**Reply by pavlenex** [reply]
I am not sure, I've never used LTC.
